admin/resources/js/components/PoolDashboard.tsx

```diff
@@ -227,12 +230,33 @@ const routes: Routes<RouterResult> = [
       component: <UpdateDepartment departmentId={params.id as string} />,
     }),
   },
+  {
+    path: searchRequestTablePath(),
+    action: () => ({
+      component: <SearchRequestPage />,
+    }),
+  },
 ];
 
 export const PoolDashboard: React.FC = () => {
   const intl = useIntl();
 
   const menuItems = [
+    <MenuHeading
+      key="search-requests"
+      text={intl.formatMessage({
+        defaultMessage: "Requests",
+        description: "Label displayed on the requests menu item.",
+      })}
+    />,
+    <MenuLink
+      key="all-requests"
+      href={searchRequestTablePath()}
+      text={intl.formatMessage({
```

Comment:
Is there a reason this isn't in the `messages` const on line 60?

Reply:
Oh ya, so should I move each message, in messages, to where they're being referenced.. or just move these into messages? I think the former.